ui/src/app/pages/data/data-set/dataset-page.spec.tsx

```diff
@@ -213,28 +213,28 @@ describe('DataSetPage', () => {
     expect(checkedValuesList.length).toBe(5);
   });
 
-  // TODO disabling temporarily for RW-7657, will refactor and enable with RW-7752
-  // it('should display correct values on rapid selection of multiple domains', async () => {
-  //   const wrapper = component();
-  //   await waitOneTickAndUpdate(wrapper);
-  //   await waitOneTickAndUpdate(wrapper);
-  //
-  //   // Select "Condition" and "Measurement" concept sets.
-  //   const conceptSetEls = wrapper.find(
-  //     '[data-test-id="concept-set-list-item"]'
-  //   );
-  //   conceptSetEls.at(0).find('input').first().simulate('change');
-  //   conceptSetEls.at(1).find('input').first().simulate('change');
-  //   await waitOneTickAndUpdate(wrapper);
-  //   await waitOneTickAndUpdate(wrapper);
-  //
-  //   const valueListItems = wrapper.find('[data-test-id="value-list-items"]');
-  //   const checkedValuesList = valueListItems.filterWhere(
-  //     (value) => value.props().checked
-  //   );
-  //   expect(valueListItems.length).toBe(5);
-  //   expect(checkedValuesList.length).toBe(5);
-  // });
+  it('should display correct values on rapid selection of multiple domains', async () => {
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    await waitOneTickAndUpdate(wrapper);
+
+    // Select "Condition" and "Measurement" concept sets.
+    const conceptSetEls = wrapper.find(
+      '[data-test-id="concept-set-list-item"]'
+    );
+    conceptSetEls.at(0).find('input').first().simulate('change');
+    await waitOneTickAndUpdate(wrapper);
```

Comment:
looks like this is the only change, plus the same below.  I don't have the context here, but this change seems reasonable.  LMK if I missed anything.

Reply:
Yes, those were the only 2 two changes other than uncommenting. Thanks!